docs/user-guide/queries.md

```diff
@@ -44,6 +45,10 @@ Answers to queries can be
 - a vector of strings delimited by `';'` `[answer1;answer2;answer3]` \[sv\]
 - a JSON string \[JSON\]
 
+## sequencing_mode
+
+As of HELICS 2.7.0 Queries have an optional parameter to describe a sequencing mode. There are currently two modes fast `helics_sequencing_mode_fast` which travels along priority channels and is identical to previous versions which all queries traveled along those channels. The other mode is `helics_sequencing_mode_ordered` which travels along lower priority channels but is ordered with all other messages in the system. This can be useful in some situations where you want previous messages to be acknowledged as part of the federation before the query is run. The `global_flush` query is forced to run in ordered mode at least after it gets to the specified target.
```

Comment:
```suggestion
As of HELICS 2.7.0 Queries have an optional parameter to describe a sequencing mode. There are currently two modes, `helics_sequencing_mode_fast` which travels along priority channels and is identical to previous versions in which all queries traveled along those channels. The other mode is `helics_sequencing_mode_ordered` which travels along lower priority channels but is ordered with all other messages in the system. This can be useful in some situations where you want previous messages to be acknowledged as part of the federation before the query is run. The `global_flush` query is forced to run in ordered mode at least until after it gets to the specified target.
```